main.tf

```diff
@@ -65,3 +76,26 @@ resource "aws_s3_bucket_public_access_block" "public_access_block" {
   restrict_public_buckets = true
 }
 
+resource "aws_s3_bucket_inventory" "inventory" {
```

Comment:
Does it make sense to default encryption with `sse_s3` ?

Reply:
The bucket is encrypted. Do we need to encrypt the file?

Reply:
yeah, nevermind :)